pkg/scheduler/objects/application.go

```diff
@@ -42,6 +42,7 @@ import (
 var (
 	reservationDelay = 2 * time.Second
 	startingTimeout  = 5 * time.Minute
+	waitingTimeout   = 5 * time.Minute
```

Comment:
5 minutes is too long, which means a job finishes and the user needs to wait for 5 minutes until they see the job is completed.  I think we should make this more aggressive, e.g 30s.  Otherwise, users will continue to complain about the issue.

Reply:
changed to 30s